policy/support/obj_perm_sets.spt

```diff
@@ -267,7 +267,7 @@ define(`relabel_chr_file_perms',`{ getattr relabelfrom relabelto }')
 # Use (read and write) terminals
 #
 define(`rw_inherited_term_perms', `{ getattr lock read write append ioctl }')
-define(`rw_term_perms', `{ rw_inherited_term_perms open }')
+define(`rw_term_perms', `{ rw_inherited_term_perms open setattr }')
```

Comment:
setattr should get own permission macro and also interfaces, like dir or files classes have.